### PR TITLE
Update README.rst

### PR DESCRIPTION
**Pull request by otmezger**
update on how to use the argument `S3OutputLocation`. took me a while to find out, so having it explicitly on the readme with an example can't hurt.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -199,6 +199,28 @@ For details of the JDBC driver options refer to the official documentation.
...
+
+    from pyathenajdbc import connect
+
+    conn = connect(s3_staging_dir='s3://YOUR_S3_BUCKET/path/to/',
+                   region_name='us-west-2',
+                   LogPath='/path/to/pyathenajdbc/log/',
+                   LogLevel='6',
+                   S3OutputLocation='s3://YOUR_S3_BUCKET/path/to/query_results/' )
+
+For details of the JDBC driver options refer to the official documentation.
```
Comment (laughingman7743) [inline README.rst]: The following link is not a document of the JDBC driver options.

File: README.rst [hunk 2]
```diff
@@ -199,6 +199,28 @@ For details of the JDBC driver options refer to the official documentation.
 
 NOTE: Option names and values are case-sensitive. The option value is specified as a character string.
 
+Specify the Query Results
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you want to specify where athena stores the `txt` and `txt.metadata` files containing the result of each query, you can specify it as follows:
```
Comment (laughingman7743) [inline README.rst]: https://docs.aws.amazon.com/athena/latest/ug/querying.html
> A results file stored automatically in a CSV format (*.csv), and
> An Athena metadata file (*.csv.metadata). 

Is it txt? Or is it csv?

File: README.rst (hunk 1 above)
Comment (otmezger) [reply README.rst]: ah yes, sorry. that was copy paste from above.

File: README.rst (hunk 2 above)
Comment (otmezger) [reply README.rst]: you know, that's also partially right. sometimes we had txt and sometimes we had csv. maybe it depends on the query...

**Comment by laughingman7743** [comment]
@otmezger Thanks!

**Pull request merged**